lsp-bridge-call-hierarchy.el

```diff
@@ -0,0 +1,172 @@
+           (get-buffer buffer)
+           :poshandler #'posframe-poshandler-frame-center
+           (list
+            :max-height posframe-height
+            :min-height posframe-height
+            :min-width  posframe-width
+            :max-width  posframe-width
+            :border-width 2
+            :border-color "gray"
+            :accept-focus t
+            ))))
+
+(defun lsp-bridge-call-hierarchy-format-call-string (s l)
+  (if (> (length s) l)
+      (format "%s%s" (substring s 0 (- l (length "..."))) "...")
+    (concat s (make-string (- l (length s)) ?\s))))
+
+(defun lsp-bridge-call-hierarchy-format-call (call)
+  (let ((name (plist-get call :name))
+        (path (plist-get call :rel_path))
+        (icon (cdr (assoc (plist-get call :icon) acm-icon-alist))))
+    (format "%s %s %s\n"
+            (if icon
+                (apply #' acm-icon-build icon)
+             " ")
+            (lsp-bridge-call-hierarchy-format-call-string name lsp-bridge-call-hierarchy-name-width)
+            (lsp-bridge-call-hierarchy-format-call-string path lsp-bridge-call-hierarchy-path-width))))
+
+(defun lsp-bridge-call-hierarchy-maybe-preview ()
+  (when (> lsp-bridge-call-hierarchy-preview-height 0)
+    (other-window 1)
+    (lsp-bridge-call-hierarchy-show)
+    (recenter)
+    (other-window -1)))
+
+(defun lsp-bridge-call-hierarchy--popup (response)
+  (interactive)
+  (setq lsp-bridge-call-hierarchy--popup-response response)
+  (setq lsp-bridge-call-hierarchy--frame
+        (lsp-bridge-call-hierarchy-posframe-show (get-buffer-create "*lsp-bridge-call-hierarchy*")))
+  (select-frame-set-input-focus lsp-bridge-call-hierarchy--frame)
+  (dolist (call lsp-bridge-call-hierarchy--popup-response)
+    (insert (lsp-bridge-call-hierarchy-format-call call)))
+  (delete-backward-char 1)
+
+  (when (> lsp-bridge-call-hierarchy-preview-height 0)
+    (split-window (selected-window) nil 'down)
+    (other-window 1)
+    (set-window-text-height (selected-window)  (1- lsp-bridge-call-hierarchy-preview-height))
+    (other-window -1))
+
+  (lsp-bridge-call-hierarchy-maybe-preview)
+
+  (lsp-bridge-call-hierarchy-mode))
+
+(defun lsp-bridge-call-hierarchy--move (next)
+  (when (and (>= (+ lsp-bridge-call-hierarchy--index next) 0)
+             (< (+ lsp-bridge-call-hierarchy--index next)
+                (length lsp-bridge-call-hierarchy--popup-response)))
+      (forward-line next)
+      (move-overlay lsp-bridge-call-hierarchy--overlay (line-beginning-position) (line-end-position))
+      (setq lsp-bridge-call-hierarchy--index
+            (+ next lsp-bridge-call-hierarchy--index))
+      (lsp-bridge-call-hierarchy-maybe-preview)
+      ))
+
+(defun lsp-bridge-call-hierarchy-next ()
+  (interactive)
+  (lsp-bridge-call-hierarchy--move 1))
+
+(defun lsp-bridge-call-hierarchy-prev ()
+  (interactive)
+  (lsp-bridge-call-hierarchy--move -1))
+
+(defun lsp-bridge-call-hierarchy-quit ()
+  (interactive)
+  (posframe-delete "*lsp-bridge-call-hierarchy*"))
```

Comment:
We should save `(selected-frame)` in lsp-bridge-call-hierarchy--popup.

Then call select-frame-set-input-focus in here.

Otherwise we can't select Emacs frame after quit lsp-bridge-call-hierarchy frame.